### PR TITLE
Merge from 0.2.0

### PR DESCRIPTION
**Pull request by chupman**
Merge to master from 0.2.0. I reviewed all the PRs that caused merge conflicts before I resolved, but please verify that your PRs were merged correctly. 

The following users have commits in this merge: @robertdale, @sjudeng, @pluradj, @krlohnes, @smcquillan, @TitiHl, @jerryjch 

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by TitiHl** [comment]
Hi @chupman confirmed my change has been merged correctly in this PR. thanks

**Comment by smcquillan** [comment]
@chupman changes from my PR have been  merged correctly, thanks.

File: janusgraph-core/src/main/java/org/janusgraph/util/stats/MetricManager.java [hunk 1]
```diff
@@ -200,7 +200,11 @@ public synchronized void removeJmxReporter() {
      * Create a {@link Slf4jReporter} attached to the JanusGraph Metrics registry.
      * <p>
      * If {@code loggerName} is null, or if it is non-null but
+<<<<<<< HEAD
      * {@link LoggerFactory#getLogger(String)} returns null, then Metrics's
+=======
+     * {@link LoggerFactory#getLogger(Class loggerName)} returns null, then Metrics's
+>>>>>>> 0.2
```
Comment (pluradj) [inline janusgraph-core/src/main/java/org/janusgraph/util/stats/MetricManager.java]: missed a merge here

Comment (chupman) [reply janusgraph-core/src/main/java/org/janusgraph/util/stats/MetricManager.java]: Doh. It's been fixed now. I did a recursive grep on '>>>>>>>', '^=======$', and '<<<<<<<' to make sure that was the only one.

**Comment by pluradj** [comment]
@robertdale @krlohnes please verify the merges of your fixes. thanks.

**Comment by StevenHarlow** [comment]
I believe `janusgraph-core/src/main/java/org/janusgraph/core/attribute/GeoshapeHelper.java:54` still references GeoShapeHelper and is breaking builds with the 0.3.0 snapshot

**Comment by StevenHarlow** [comment]
Ah, nevermind. I see you address this here: https://github.com/JanusGraph/janusgraph/pull/1121

**Comment by chupman** [comment]
Sorry about the break. I'm working on adding CI coverage to prevent this particular slip up from occurring again.

**Pull request merged**